### PR TITLE
#1115 Distinguishes HC border from focused HC border, removes double stoke divider line

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [x] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

Distinguishes HC border from focused HC border 

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss [hunk 1]
```diff
@@ -36,6 +55,14 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
     @include padding(0, 0, 0, 26px);
     display: inline-block;
   }
+
+  @media screen and (-ms-high-contrast: active) {
+    margin-bottom: -1px;
```
Comment (aleksandrjPersonal) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss]: Corrects for double stroke artifact when items stack

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss [hunk 2]
```diff
@@ -313,5 +340,5 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
 }
 
 :global(.ms-Fabric.is-focusVisible) .choiceFieldIsInFocus {
-  border: 1px solid $ms-color-neutralPrimary;
+   @include highContrastBorder($ms-color-neutralPrimary);
```
Comment (aleksandrjPersonal) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss]: adds inner stroke on focus

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss [hunk 3]
```diff
@@ -14,6 +14,25 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
 //== Component: Choicefield group
 //
 
+@mixin highContrastBorder($color) {
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss]: Aren't we using this mixin elsewhere? If so, you should import it and use it instead. If we AREN'T, we probably should. I.E. Look at how this is solved in other areas.

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt Moved mixin to common file. Renamed to use xxx-Xxxx-Xxx pattern

**Comment by micahgodbolt** [comment]
@aleksandrjPersonal there doesn't appear to be a difference between @focus-border and @highcontrast-focus-border.

Can't you just use @focus-border, and pass your high contrast color in?

**Comment by aleksandrjPersonal** [comment]
Closing to target master

**Pull request closed without merging**